tests/ImageSharp.Web.Tests/Processors/ResizeWebProcessorTests.cs

```diff
@@ -168,7 +168,7 @@ public void ResizeWebProcessor_RespectsOrientation_Center(ushort orientation)
             image.Metadata.ExifProfile.SetValue(ExifTag.Orientation, orientation);
             using var formatted = new FormattedImage(image, PngFormat.Instance);
 
-            PointF expected = GetExpectedCenter(orientation, new PointF(x, y));
+            PointF expected = ExifOrientationUtilities.Transform(new Vector2(x, y), Vector2.Zero, Vector2.One, orientation);
             ResizeOptions options = ResizeWebProcessor.GetResizeOptions(formatted, commands, parser, culture);
             Assert.Equal(expected, options.CenterCoordinates);
```

Comment:
This test failed on my machine, because my decimal separator is a comma and the interpolated string above uses the current culture: `$"{x},{y}"`. This can be fixed by updating it to `FormattableString.Invariant($"{x},{y}"))` 👍🏻

Reply:
Yeah, my bad.